pkgs/by-name/sq/sql-migrate/package.nix

```
@@ -0,0 +1,38 @@
+{ stdenv
+, lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "sql-migrate";
+  version = "1.6.1";
+
+  src = fetchFromGitHub {
+    owner = "rubenv";
+    repo = "sql-migrate";
+    rev = "v${version}";
+    hash = "sha256-l9HW8L9khr+UfxlhH9z2mrzjemlqRbnnM7rPsf9SCtg=";
+  };
+
+  vendorHash = "sha256-GDM2YAqLkkmfOJShBbPpxS0Zs3zOK4yCfTCjohSGNIM=";
+
+  # Bundle release metadata
+  ldflags = [
+    # Omit the symbol table and debug information.
+    "-s"
+    # Omit the DWARF symbol table.
+    "-w"
+  ];
+
+  doCheck = true;
+
+  meta = with lib; {
+    homepage = "https://github.com/rubenv/sql-migrate";
+    description = "A SQL schema migration tool for Go programs";
```

```suggestion
    description = "SQL schema migration tool for Go programs";
```
see contributing guide